test_add_tag.py

```diff
@@ -0,0 +1,82 @@
+# import pytest
```

Comment:
If your method is designed to handle duplicates, it may be useful to test if adding the same tag twice results in only one entry or multiple entries. This could look like:
def test_duplicate_tag_handling(myList):
    myList.add_tag(1, "duplicate")
    myList.add_tag(1, "duplicate")
    task = myList.tasks[0]
    assert task[3].count("duplicate") == 1  # Adjust as needed based on desired behavior